scripts/mask_hdf.py

```diff
@@ -53,9 +62,13 @@
     mask = None
     if args.type == LST:
         mask = build_lst_mask(dat.get(), quality.get())
+        masked_dat = np.ma.array(dat.get(), mask=mask, fill_value=fill_value, dtype=DAT_NUMPY)
     elif args.type == NDVI:
         mask = build_ndvi_mask(dat.get(), quality.get())
-    masked_dat = np.ma.array(dat.get(), mask=mask, fill_value=fill_value, dtype=DAT_NUMPY)
+        masked_dat = np.ma.array(dat.get(), mask=mask, fill_value=fill_value, dtype=DAT_NUMPY)
+    elif args.type == SNOW:
+        mask = build_snow_mask(dat.get())
```

Comment:
I'm going to pull this in and resolve this issue anyway so I can merge some changes I've been holding back.